assets/styles/themes/_light.scss

```diff
@@ -283,5 +283,10 @@ $selected: rgba($primary, .5);
 
   --product-icon: #{$darker};
   --product-icon-active: #{$darkest};
+
+  // Make text muted slightly darker so it is readable
+  .text-muted {
+    color: $dark !important;
```

Comment:
It looks like this should be achieved by setting `muted:      $dark,` in `@include color-variables` above. In that mixin it creates a set of colours which are rarely used and of the styles only `text-muted` is used.... so should be a safe change